Mage/RPCClient.cs

```diff
@@ -25,8 +14,10 @@ public class RPCClient : JSONRPC {
 	public void call(string methodName, JObject parameters, Action<Exception, JToken> cb) {
 		// Attach anyre required mage headers
 		Dictionary<string, string> headers = new Dictionary<string, string> ();
-		if (_sessionKey != null) {
-			headers.Add("X-MAGE-SESSION", _sessionKey);
+		string sessionKey = mage.session.GetSessionKey ();
+
+		if (sessionKey != null) {
```

Comment:
Up to you guys how you wanna define your C# style, but I'm a big fan of String's `isNullOrEmpty`.